src/Api.php

```diff
@@ -206,33 +212,81 @@ public function delete($pattern, $callable = null)
     /**
      * Implement REST pattern matching.
      *
-     * @param string           $pattern
-     * @param \atk4\data\Model $model
+     * @param string                    $pattern
+     * @param \atk4\data\Model|callable $model
      *
      * @return mixed
      */
     public function rest($pattern, $model = null)
     {
+        // GET all records
         $this->get($pattern, function () use ($model) {
+            $args = func_get_args();
+
+            if (is_callable($model)) {
```

Comment:
wouldn't that be handled by $this->get() ? perhaps it should be?

Reply:
by get() no, but probably could be done somehow in match() method.

Reply:
yeah, that would be great.